testapp/src/main/res/layout/settings_menu_activity.xml

```diff
@@ -71,7 +71,58 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/small_12"
         android:text="@string/lb_test_mode"
-        android:visibility="invisible"/>
+        android:visibility="gone"/>
+
+  </LinearLayout>
+
+  <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical"
+      app:layout_constraintStart_toStartOf="@id/toolbar"
+      app:layout_constraintBottom_toBottomOf="parent">
```

Comment:
I think these buttons should be directly beneath the input fields (maybe with some margin between the two) instead of constrained to the bottom of the view. It seems difficult to notice the buttons when they're at the bottom. Also, this would more closely match the iOS version. What do you think?

Reply:
Yeah, I have updated the UX with screenshot in PR's description.